### ISSUE TITLE
How can I get a texture filename from a scene

### ISSUE THREAD

**Issue by ThomasT75** [jkvargas/russimp#10]
I'm following a [OpenGL tutorial](https://learnopengl.com/Model-Loading/Model) but after a while trying to figure out how to get a texture filename all I can get from the scene is a part of the filename 

This is what I got so far 

> Note: this is the [obj](https://learnopengl.com/data/models/backpack.zip) I'm using is from the tutorial

```
let scene = Scene::from_file("path/to/obj",
      vec![
          PostProcess::Triangulate,
          PostProcess::FlipUVs,
          PostProcess::EmbedTextures
      ]      
).unwrap();

let material = &scene.materials[1];

for i in 0..material.0.len() {
    match &material.0[i].key[..] {
        "$tex.file" => println!("{:?}", std::str::from_utf8(&material.0[i].data).unwrap()),
        _ => {},
    }
}

```
Prints:
```
"\u{b}\u{0}\u{0}\u{0}*0\u{0}fuse.jpg\u{0}"
"\u{c}\u{0}\u{0}\u{0}*1\u{0}cular.jpg\u{0}"
"\n\u{0}\u{0}\u{0}*2\u{0}mal.png\u{0}"
```

> Note: I can get only the letters by converting only this range [7..len - 1] but is not the full filename

the full texture filenames are 

- diffuse.jpg
- specular.jpg
- normal.png

also the scene.textures is like this:
```
[
    Texture {
        filename: "",
        height: 0,
        width: 6108855,
        ach_format_hint: "jpg",
        data: [],
    },
    Texture {
        filename: "",
        height: 0,
        width: 6722296,
        ach_format_hint: "jpg",
        data: [],
    },
    Texture {
        filename: "",
        height: 0,
        width: 15214871,
        ach_format_hint: "png",
        data: [],
    },
]
```
Idk if I'm doing something wrong or the data is just like that, but any help is appreciated :D 
also Thanks for porting this to rust :+1:

**Comment by jkvargas** [comment]
Hey dude.
I am doing some refactors on the material side.
This week I will release it and I will let you know about it :)

**Comment by jkvargas** [comment]
Hey dude, I am releasing version 1.8 Today.
I did some refactors on the material side.
the property info will be delivered with the wanted data instead of a byte array.
I was able to glance at your issue.
made a small test inside the crate with your file locally and the filenames return correctly, doing a small application referencing the crate return those weird characters. I will continue to investigate it during the week.

`pub enum PropertyTypeInfo {
    Buffer(Vec<u8>),
    IntegerArray(Vec<i32>),
    FloatArray(Vec<f32>),
    String(String),
}`
